okta/UsersClient.py

```diff
@@ -8,8 +8,10 @@
 
 
 class UsersClient(ApiClient):
-    def __init__(self, base_url, api_token):
-        ApiClient.__init__(self, base_url + '/api/v1/users', api_token)
+    def __init__(self, *args, **kwargs):
+        kwargs['base_url'] = (kwargs['base_url'] or args[0]) + '/api/v1/users'
```

Comment:
We should check for `args` length and/or `base_url`/ `api_token` here. Then throw an error.

```python
if not 'base_url' in kwargs and len(args) < 1:
    raise ValueError('Missing args for UsersClient')
```